src/main/webapp/user-page.html

```diff
@@ -31,6 +31,16 @@
     </nav>
     <h1 id="page-title">User Page</h1>
 
+    <!-- This creates an about me text placeholder in the logged user's page. -->
+    <div id="about-me-container">Loading...</div>
+    <div id="about-me-form" class="hidden">
+      <form action="/about" method="POST">
+        <textarea name="about-me" placeholder="about me" rows=4 required></textarea>
+        <br/>
```

Comment:
Instead of `<br/>`, I would recommend putting `<div>` elements around the pieces that you want to have on separate lines. Using `<div>` instead of `<br>` makes it easier to control the margin/padding and other aspects of the display through CSS.

In other words:

     <form ...>
      <div class="form-contents">
          <div class="input-container">
             <textarea ...>
          </div>
          <div class="input-container">
             <input ... >
          </div>
      </div>
     </form>

This will already put them on separate lines. If you want to adjust spacing between them, you can use something like the following CSS code:

       /* ... */
       .input-container {
          margin-top: 8px;
       }

       .input-container:first-child {
          margin-top: 0;
       }
       /* ... */